packages/scandipwa/src/component/ProductPrice/ProductPrice.style.scss

```diff
@@ -13,14 +13,15 @@
     --price-color: #{$black};
     --price-with-discount-color: #b10d0d;
     --price-discount-color: #808080;
```

Comment:
is is actually used anywhere? I see 2 cases. HighPrice and RegularPrice elements. HighPrice element is present in component, but RegularPrice I don't see. Can you please check if there is any render whit this element and if no, then you can actually just change value in this variable instead of creating new one + delete that one &-ReguralPrice styles.